build/checkstyle/tachyon_checks.xml

```diff
@@ -217,6 +217,14 @@
     </module>
     <module name="MissingDeprecated"/>
     <module name="EqualsHashCode"/>
+    <!--
```

Comment:
Changed the PR accordingly. I will add it in the last PR to this topic after the preparation work is done.